R/getPrevalence.R

```diff
@@ -148,32 +139,28 @@
 #' prevalent <- getPrevalent(tse,
 #'                             rank = "Phylum",
 #'                             detection = 10,
-#'                             prevalence = 50/100,
-#'                             as.relative = FALSE)
+#'                             prevalence = 50/100)
 #' head(prevalent)
 #'
 #' # Gets a subset of object that includes prevalent taxa
 #' altExp(tse, "prevalent") <- subsetByPrevalent(tse,
 #'                                              rank = "Family",
 #'                                              detection = 0.001,
-#'                                              prevalence = 0.55,
-#'                                              as.relative = TRUE)
+#'                                              prevalence = 0.55)
```

Comment:
Now this detection threshold is wrong (the values are counts, but this is relative). You could add

assay.type = "relabundance"